middleware/authorization.js

```diff
@@ -6,18 +6,19 @@ const bcrypt = require("bcrypt");
 
 async function issueToken (req, res) {
     const { body } = req;
-    const row = await userRepository.findUser_email(body.email);
-    const tutee = await tuteeRepository.findTuteeId(row.id);
+    const row = await userRepository.findUserEmail(body.email);
 
     if (!row) {
-        return res.status(400).json( {message: "해당 이메일이 없습니다."});
+        return res.status(400).json({ message: "해당 이메일이 없습니다." });
```

Comment:
```suggestion
        return res.status(404).json({ message: "해당 이메일이 없습니다." });
```
찾으려는 데이터가 없을때는 404 NOT FOUND입니다.